bing-dict.el

```diff
@@ -133,8 +133,52 @@ The value could be `synonym', `antonym', `both', or nil.")
                                          'face
                                          'font-lock-builtin-face))
 
+(defvar bing-dict--org-file "~/.orgs/vocabulary.org"
```

Comment:
The default path should be something consisting of `user-emacs-directory` and `bing-dict`. Say "~/.emacs.d/bing-dict/".